### DISCUSSION TITLE
How can I limit the amount of signals being sent

### DISCUSSION THREAD

**Discussion by thow21** [TelemetryDeck/Viewer#269] [Q&A]
I am currently working on an app project in kotlin that uses a lot of activities. I get a signal for every lifecycle method of every activity. I also send my own signals. How can I get rid of all lyfecycle signals?

**Comment by winsmith** [comment]
Sorry for the late reply!

You can set `TelemetryManager.configuration.sendNewSessionBeganSignal` to `false`, which will disable the automatic sending of the NewSessionBegan signal. This will reduce the number of signals.
